affirm_example/__init__.py

```diff
@@ -53,6 +53,12 @@ def capture_charge(charge_id, amount=None):
                          **_get_extra_request_args()).json()
 
 
+def originate_charge(charge_id):
+    originate_charge_url = "{0}/charges/originate/{1}".format(app.config["AFFIRM"]["NONPROD_API_URL_BASE"], charge_id)
```

Comment:
how does the config get set in the example app? `NONPROD_API_URL_BASE` isn't in the config from what I can see.

Reply:
See https://phabricator.team.affirm.com/D12848